ci/build_docs.sh

```diff
@@ -0,0 +1,36 @@
+set -euo pipefail
```

Comment:
```suggestion
#!/bin/bash
set -euo pipefail
```
We should add this [shebang](https://linuxize.com/post/bash-shebang/) back to ensure the script works correctly